src/functTest/groovy/com/bmuschko/gradle/docker/DockerRemoteApiPluginFunctionalTest.groovy

```diff
@@ -114,7 +115,7 @@ class DockerRemoteApiPluginFunctionalTest extends AbstractGroovyDslFunctionalTes
             import com.bmuschko.gradle.docker.tasks.image.DockerPullImage
             import com.bmuschko.gradle.docker.tasks.image.DockerRemoveImage
             import com.bmuschko.gradle.docker.tasks.image.Dockerfile
-            
+
```

Comment:
auto formatting, is this OK to remove @bmuschko?

Reply:
That's fine. Seem like a tab or tabs ended up here.